include/rocksdb/slice.h

```diff
@@ -44,11 +41,9 @@ class Slice {
   /* implicit */
   Slice(const std::string& s) : data_(s.data()), size_(s.size()) {}
 
-#ifdef __cpp_lib_string_view
   // Create a slice that refers to the same contents as "sv"
   /* implicit */
   Slice(std::string_view sv) : data_(sv.data()), size_(sv.size()) {}
```

Comment:
Should this pass in a const std::string_view& ?

Reply:
OK